### DISCUSSION TITLE
Custom logo help

### DISCUSSION THREAD

**Discussion by dominic-wood** [sct/overseerr#974] [Help] [answered]
I know that this feature was included in release 1.19 and I may be being completely oblivious, but I can't see where to use my own logo in place of the default Overseerr one?

**Comment by TheCatLady** [comment, marked as answer]
Hi @jobbieontheknee! Custom logo functionality is planned but has not been added yet.  You can track #404 for updates.

**Reply by dominic-wood** [reply]
Thanks for the update @TheCatLady!

**Comment by WoisWoi** [comment]
Hi !

Just so you know : If you don't want to wait, you can always change them by yourself in the public folder. ;)

**Reply by hamoudydev** [reply]
How would you go about doing this? I  see the public folder on github but not in the docker. Is it hidden?

**Reply by RedSpid3r** [reply]
can easily do this in docker, simple mount the following:
Container Path: `/app/public/logo.png` -> Host Path: `/path/to/custom/logo.png`

**Reply by hamoudydev** [reply]
Thank you, seems that the reason I'm not seeing the option to edit the logo is because I am on unraid.

**Reply by RedSpid3r** [reply]
Here is my Unraid docker config for custom logo:
![image](https://user-images.githubusercontent.com/12657277/126164273-91cf85f1-e524-447b-8e4f-dd97f30ac943.png)

**Reply by WoisWoi** [reply]
> 
> 
> Thank you, seems that the reason I'm not seeing the option to edit the logo is because I am on unraid.

Hi,
There is no option to edit the logo(s) via Overseerr -yet-.
As RedSpid3r just said, you have to map your logo path to the Overseerr logo path.

**Reply by hamoudydev** [reply]
> 
> 
> Here is my Unraid docker config for custom logo:
> ![image](https://user-images.githubusercontent.com/12657277/126164273-91cf85f1-e524-447b-8e4f-dd97f30ac943.png)

Yup, this is what I did and no luck. Assumed you were talking about the docker container rather than the one in unraid since the github repository looks different. I am using the linuxserver repository and I don't see a logo path in appdata. Tried the other two containers available in the community apps as well with the same result. 

EDIT: got it to work by using hotio:latest. Doesn't work for me on any of the linuxserver. Previously I was trying it on hotio:develop.

THANK YOU!!!!

**Reply by tscibilia** [reply]
incase anyone is using linuxserver repo, you need to use a slightly different path to get this to work properly
/mnt/user/appdata/overseerr/logo.png:/app/overseerr/public/logo.png

**Comment by hamoudydev** [comment]
Did this break for anyone with the latest (1.26.x) update?

**Reply by RedSpid3r** [reply]
Yes because the original path changed. 

You need to update the container path to:
`/app/public/logo_full.svg`

**Comment by WoisWoi** [comment]
You can find everything in there :
https://github.com/sct/overseerr/tree/develop/public

Le lun. 20 sept. 2021 à 14:15, RedSpid3r ***@***.***> a
écrit :

**Reply by hamoudydev** [reply]
Hey thanks! I tried everything but the .svg extension before posting.

**Reply by mosteanuv** [reply]
Hey. If you have time could you please help me with an answer on what I posted lower?
Thanks.

**Comment by mosteanuv** [comment]
Hey guys. I have tried this but I get an icon that shows a crash icon instead of the logo.

![set](https://user-images.githubusercontent.com/94706076/223882521-3801bf25-b361-4610-b61e-b09b173c1758.jpg)
![crash](https://user-images.githubusercontent.com/94706076/223882654-2fc872aa-cd24-448e-aaad-1333a651fa9e.jpg)

Please help!

**Reply by WoisWoi** [reply]
Hi,

It's the wrong extension. The source is .svg and your logo is .png ;)

**Reply by mosteanuv** [reply]
I don t know why I thought you can replace them. Thanks a lot!!

**Comment by BenH4** [comment]
Did anyone manage to set an different logo recently or is it still working for the others? I am having trouble in my case... I am using binhex repo with my unraid server. I found that the container path should be "/app/overseerr/public/logo_full.svg" with this repo (I tried without "overseerr" as well). What is strange is that I don't even have the crash logo as some other people, the standard overseerr logo is still being used. 
Does anyone have an idea?
Here is the path config: 
 
![Capture d’écran 2023-12-22 231902](https://github.com/sct/overseerr/assets/32013305/8d3d407a-1a03-44de-91a1-c4b73213b2f1)

**Reply by WoisWoi** [reply]
Hi,

Did you try to either clear the cache of your web browser or to use a different browser ?
And you're sure that your .svg logo is "working" ?

P.S : And yes, it's still working on my side (with the last stable version).

**Reply by BenH4** [reply]
Hi,
Thanks for our response.
I have tried both clearing the cache and use another browser, nothing changed, the overseer logo is still used.
For the svg file, it was made with Photoshop/illustrator and I can open it in the web browser, so I guess it is working.

EDIT: Never mind, it was my mistake, actually with the binhex repo, the file was elsewhere. For those looking for it: /usr/lib/overseerr/public/logo_full.svg
